manim/mobject/geometry.py

```diff
@@ -2233,6 +2237,8 @@ def __init__(
         color=WHITE,
         height=2.0,
         width=4.0,
+        grid_xstep=0,
```

Comment:
If I want more grid lines I put the step value as negative numbers?

Reply:
No. You say what spacing you want. If you say `grid_xstep=0.5`, this will give you 8 vertical lines, all 0.5 units apart.